protocol/contracts/amo/test/RamosTestnetTempleTokenVault.sol

```diff
@@ -3,25 +3,37 @@ pragma solidity ^0.8.17;
 // Temple (amo/test/RamosTestnetTempleTokenVault.sol)
 
 import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
-import { IRamosProtocolTokenVault } from "contracts/interfaces/amo/helpers/IRamosProtocolTokenVault.sol";
+import { IRamosTokenVault } from "contracts/interfaces/amo/helpers/IRamosTokenVault.sol";
 import { ITempleERC20Token } from "contracts/interfaces/core/ITempleERC20Token.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
Not a big deal, but I tend to put all the 3rd party libs together at the top of the imports